modules/javafx.controls/src/main/java/javafx/scene/control/TableColumnBase.java

```diff
@@ -726,42 +726,42 @@ public final T getCellData(final S item) {
      */
     public abstract ObservableValue<T> getCellObservableValue(S item);
 
-    /** {@inheritDoc} */
-    @Override public EventDispatchChain buildEventDispatchChain(EventDispatchChain tail) {
+    @Override
+    public EventDispatchChain buildEventDispatchChain(EventDispatchChain tail) {
         return tail.prepend(eventHandlerManager);
     }
 
     /**
-     * Registers an event handler to this table column. The TableColumnBase class allows
-     * registration of listeners which will be notified when editing occurs.
-     * Note however that TableColumnBase is <b>not</b> a Node, and therefore no visual
-     * events will be fired on it.
-     *
-     * @param <E> The type of event
-     * @param eventType the type of the events to receive by the handler
-     * @param eventHandler the handler to register
-     * @throws NullPointerException if the event type or handler is null
+     * {@inheritDoc}
+     * <p>
+     * The {@code TableColumnBase} class allows registration of listeners which will be notified when editing occurs.
```

Comment:
The edit events are defined only in the subclasses. Either, if possible, move the events to this class, or clarify that these events are defined in subclasses. `TreeItem` talks more about this - using the cell.